sdks/go/pkg/beam/runners/prism/internal/engine/elementmanager.go

```diff
@@ -1620,6 +1641,13 @@ func (ss *stageState) startEventTimeBundle(watermark mtime.Time, genBundID func(
 	}
 
 	bundID := ss.makeInProgressBundle(genBundID, toProcess, minTs, newKeys, holdsInBundle)
+	if _, ok := ss.kind.(*aggregateStageKind); ok {
+		// For aggregate stage, buildEventTimeBundle may have saved bundle panes with empty string as key.
+		// Move it under the correct BundleID now.
+		ss.bundlePanes[bundID] = ss.bundlePanes[""]
```

Comment:
We could probably avoid the sneaky empty-string handling by returning a `panesInBundle` map from buildEventTimeBundle, like we are doing for `holdsInBundle`.  Similar "refused bequests" to non-stateful stages needing to return an empty holds map or empty newKeys.